MODELGUIDE.md

```diff
@@ -224,7 +224,7 @@ Flow.Ipv4:
       x-field-pattern:
         format: ipv4
         default: 0.0.0.0
-        count: false
+        features: [count, random]
```

Comment:
According to docs `x-device-pattern` has similar features. So, `random` should applicable for `x-device-pattern` though I can't find this `x-device-pattern` pattern within test. Please extend test coverage if `x-device-pattern` present within bundler code.

Reply:
I dont think we use x-device pattern any more , its been removed